src/main/java/org/reactome/release/dataexport/EuropePMCFileUploader.java

```diff
@@ -0,0 +1,155 @@
+package org.reactome.release.dataexport;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Class for updating Reactome export files on the EuropePMC FTP Server.  This class will connect to the EuropePMC FTP
+ * Server on instantiation and provides methods to:
+ *
+ * 1) Upload new (i.e. current Reactome release) profile & links files, created by the EuropePMC class
+ * 2) Delete old (i.e previous Reactome release) profile & links files
+ * 3) List files present on the EuropePMC FTP Server for confirmation of successful file upload/deletion.
+ *
+ * @author jweiser
+ * @see EuropePMC
+ */
+public class EuropePMCFileUploader extends FTPFileUploader {
+	private static final List<String> EUROPE_PMC_SPECIFIC_REQUIRED_PROPERTIES = Arrays.asList(
+		"europePMCFTPUserName",
+		"europePMCFTPPassword",
+		"europePMCFTPHostName",
+		"europePMCFTPReactomeFolderPath"
+	);
+
+	/**
+	 * Returns a new instance of this class responsible for uploading files to the EuropePMC FTP Server.
+	 *
+	 * The properties files must have keys for the following or an IllegalStateException will be thrown:
+	 * 1. reactomeVersion - the current release version for Reactome
```

Comment:
We use `releaseNumber` in the configuration files used by the release pipelines

Reply:
Okay, so I'll change the name here to match?

Reply:
Sounds good! Will probably want to do a project-wide search/update

Reply:
Done: https://github.com/reactome/release-data-exporter/pull/4/commits/cad8257d96b4f7494b13836b0842c2a743aac43f